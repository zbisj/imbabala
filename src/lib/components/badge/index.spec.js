import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";

import Badge from ".";

describe("Badge", () => {
  test("renders correctly", () => {
    const tree = renderer
      .create(<Badge label="test" pill={true} variant="primary" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
