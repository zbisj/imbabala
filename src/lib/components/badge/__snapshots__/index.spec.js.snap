// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Badge renders correctly 1`] = `
<p
  className=""
>
  test
</p>
`;
